### ISSUE TITLE
[id] attributes on active, focusable elements are not unique

### ISSUE THREAD

**Issue by katydecorah** [mapbox/dr-ui#389]
**Describe the bug**

In PageLayout, the Feedback component is included twice: in the sidebar and bottom of page. Using CSS, we hide and show one instance or the other based on device width. Feedback includes a bunch of `id` attributes to help with testing, but since both instances are technically rendered each instance `id`s are not unique. As a result the Lighthouse accessibility report returns the following:

```
[id] attributes on active, focusable elements are not unique
Failing Elements
--
button#dr-ui--feedback-page-yes.btn.btn--s
button#dr-ui--feedback-page-no.btn.btn--s.ml6
```

This was a known bug during the info arch work, but I never ticketed or followed up.

**To Reproduce**

1. Go to https://docs.mapbox.com/studio-manual/examples/3d-buildings/
2. Open Console then go to Lighthouse
3. Run accessibility report.

**Expected behavior**

The `id` attributes are not required for the Feedback component to run. We can replace with `data-test` attribute or add a `position` prop to help make the `id` more unique.
